homeassistant/components/onboarding/views.py

```diff
@@ -201,9 +202,7 @@ async def post(self, request, data):
                     "invalid client id or redirect uri", HTTP_BAD_REQUEST
                 )
 
-            # Return authorization code so we can redirect user and log them in
```

Comment:
We need this to be logged in after onboarding. We should not remove it.

Reply:
From what I can gather it was removed here from what I can gather: https://github.com/home-assistant/frontend/commit/cc71ccaafa7c4e95c646a0bb23d7a568568d74ae

Reply:
It's still in use: https://github.com/home-assistant/frontend/blob/dev/src/onboarding/ha-onboarding.ts#L275

Reply:
I think i have resolved this. I'm trying to decide if remembering the credential of the request is a good thing.